setup.py

```diff
@@ -4,9 +4,12 @@
 with open('requirements.txt') as f:
     requirements = f.read().splitlines()
 
+# Read in the version number
+exec(open('axelrod/version.py', 'r').read())
+
```

Comment:
I tried that but it still trips up on the dependencies:

```
(tmp) Axelrod(731) ✗: python setup.py install
Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    from axelrod.version import __version__
  File "/Users/vince/src/Axelrod/axelrod/__init__.py", line 9, in <module>
    from .random_ import random_choice, seed
  File "/Users/vince/src/Axelrod/axelrod/random_.py", line 2, in <module>
    import numpy
ImportError: No module named 'numpy'
(tmp) Axelrod(731) ✗: cat setup.py
from setuptools import setup

# Read in the requirements.txt file
with open('requirements.txt') as f:
    requirements = f.read().splitlines()

# Read in the version number
#exec(open('axelrod/version.py', 'r').read())
from axelrod.version import __version__

setup(
    name='Axelrod',
    version=__version__,
    install_requires=requirements,
    author='Vince Knight, Owen Campbell, Karol Langner, Marc Harper',
    author_email=('axelrod-python@googlegroups.com'),
    packages=['axelrod', 'axelrod.strategies', 'axelrod.tests'],
    url='http://axelrod.readthedocs.org/',
    license='The MIT License (MIT)',
    description='Reproduce the Axelrod iterated prisoners dilemma tournament',
)
```

The `exec` approach is suggested here because of this issue: http://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package